fast_matched_filter/src/matched_filter.c

```diff
@@ -70,7 +71,7 @@ void matched_filter(float *templates, float *sum_square_templates, int *moveouts
                                                         n_samples_template,
                                                         n_samples_data,
                                                         n_stations,
-                                                        n_components);
+                                                        n_components, i);
```

Comment:
This new extra argument is no longer necessary if we go for the conservative approach, right?